votes/0146-sig-migrate-reviewer.md

```diff
@@ -0,0 +1,23 @@
+# A Vote for @3pointer and @glorv as TiKV Committer
+
+## Proposal
+
+Add @3pointer and @glorv as TiKV Committer, all of whom are committers of the previous SIG-Migrate.
+
+@3pointer has contributions to the Backup and Restore project.
+* [Authored pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+author%3A3pointer)
+
+@glorv has contributions to the TiDB-Lightning project.
+* [Authored pull requests](https://github.com/tikv/tikv/pulls?q=is%3Apr+author%3Aglorv)
```

Comment:
I can see most contributions are bug fix and small improvements. Some key contributions like designs or implement of major features seems more qualified to me.

Reply:
Beside designs or features, we should also consider the maintenance, they are the de facto maintainers for backup and sst-importer components.

Reply:
If someone is assigned to maintain a module, then they need to show their qualification by contributions. We can't say they will be de facto maintainers because they are asked to or appointed by job requirements.

Reply:
@overvenus could you list the activities they make on traiging issues or reviewing significant changes?